src/wrapt/_wrappers.c

```diff
@@ -127,7 +127,7 @@ static int WraptObjectProxy_init(WraptObjectProxyObject *self,
 {
     PyObject *wrapped = NULL;
 
-    static char *kwlist[] = { "wrapped", NULL };
+    char *const kwlist[] = { "wrapped", NULL };
```

Comment:
FWIW. Changing this to `const` instead of `static` will result in the compiler complaining (but not erroring), with:

```
src/wrapt/_wrappers.c:2316:45: warning: passing 'char *const[8]' to parameter of type 'char **' discards qualifiers [-Wincompatible-pointer-types-discards-qualifiers]
 2316 |             "OOO|OOOO:FunctionWrapperBase", kwlist, &wrapped, &instance,
 ```

since Python doesn't use `const` in the prototype of `PyArg_ParseTupleAndKeywords` and similar functions.